crypto/CMakeLists.txt

```diff
@@ -747,6 +747,8 @@ if(BUILD_TESTING)
   target_link_libraries(${CRYPTO_TEST_EXEC} test_support_lib boringssl_gtest crypto)
   if(WIN32)
     target_link_libraries(${CRYPTO_TEST_EXEC} ws2_32)
+  else()
+    target_compile_options(${CRYPTO_TEST_EXEC} PUBLIC -Wno-deprecated-declarations)
```

Comment:
I assume there is no way to have a suppression file of sorts? Otherwise kind of annoying we need to turn off the warning completely.